mimicpy/parsers/mpt.py

```diff
@@ -139,17 +141,25 @@ def __translate(self, selection):
            selection eg., resname is SER and id < 25 and mol not Protein_chain_B
            will be translated to np_vals['resname'] == 'SER' and np_vals['id'] < 25 and np_vals['mol'] != 'Protein_chain_B'
          """
-    
+         
+        selection = selection.replace('(', ' ( ').replace(')', ' ) ') # put space between brackets
```

Comment:
What do you mean? This line just changes ')or' to ') or', because if there is no space then it causes a problem for the parser. I don't think a function is necessary for one line.

Reply:
```suggestion
        selection = space_brackets(selection)
        
        def space_brackets(string)
            string = string.replace('(', ' ( ').replace(')', ' ) ')
            return string
```

Reply:
A full function seems like overkill, since it will only be used once in the function. I made it a lambda function instead. I will resolve this now, but feel free to change it if you want while refactoring.